src/ui/components/OverlayCard/OverlayCard.scss

```diff
@@ -1,8 +1,9 @@
 @import "~core/css/inc/vars";
 
 .OverlayCard {
-  margin: 0 $padding-page;
-  width: 100%;
+  margin: 0;
+  width: 90%;
```

Comment:
Why this change? We tend not to use `%` widths and instead `px` margins to keep the margins in-line with the mocks.